src/pretix/control/forms/item.py

```diff
@@ -627,7 +627,9 @@ class Meta:
                 'class': 'scrolling-multiple-choice'
             }),
             'generate_tickets': TicketNullBooleanSelect(),
-            'show_quota_left': ShowQuotaNullBooleanSelect()
+            'show_quota_left': ShowQuotaNullBooleanSelect(),
+            'max_per_order': forms.widgets.NumberInput(attrs={'min': 0}),
+            'min_per_order': forms.widgets.NumberInput(attrs={'min': 0}),
```

Comment:
I think only doing this in the widget is bad, as it will only be client side and not validated server side. The correct level in the form layer would be the `forms.IntegerField`, but even that would not affect the API etc. As discussed I don't want to use PositiveIntegerField since the migration will be dangerous, but I think we should add a [MinValueValidator](https://docs.djangoproject.com/en/4.0/ref/validators/#minvaluevalidator) to the IntegerField on the model – maybe combined with the widget (or form field) change.

Reply:
Good idea, I will add this as well!

Reply:
I added the MinValueValidator to the model-field. I also tried setting the form-field in the UpdateForm to an IntegerField with validators, but that did not translate to a min-attribute on the input[type=number]. So in the form I left the forms.widget stuff in there for the min-attribute in HTML.